src/Adapter/Platform/AbstractPlatform.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
```

Comment:
Please revert. The topic of this PR is "Delegate quoteIdentifier operation to quoteIdentifierChain" and not fixing the CS!